packages/kbn-unified-data-table/src/components/data_table.scss

```diff
@@ -1,4 +1,4 @@
-.dscDiscoverGrid {
+.udtDataTable {
```

Comment:
Should we use `.unifiedDataTable` here? to match what we're doing with `.unifiedFieldList`?

Reply:
Resolved with [commit](https://github.com/elastic/kibana/pull/163211/commits/29c14c94dc153d8299779c2979c83d83cbfa8e5b)